src/mono/wasm/polyfills/msrcrypto.js

```diff
@@ -0,0 +1,9404 @@
+//*******************************************************************************
+//
+//    Copyright 2020 Microsoft
```

Comment:
This should get mention in https://github.com/dotnet/runtime/blob/master/THIRD-PARTY-NOTICES.TXT

Reply:
addressed in recent commits